homeassistant/components/xiaomi_miio/sensor.py

```diff
@@ -174,16 +167,6 @@ def __init__(self, name, device, model, unique_id):
             ATTR_MODEL: self._model,
```

Comment:
We should create one entity per measurement instead of putting everything in the state attributes. Use as many appropriate device classes as possible:
https://developers.home-assistant.io/docs/core/entity/sensor#available-device-classes

The static info, like model, should be removed.

Reply:
I removed the model as it is already in the device_info.

Splitting up in seperate sensor entities is needed across the whole Xiaomi Miio platform, this PR is one of the steps in getting to that goal (it is actually the reason why I started converting everythin to config flow).
The whole conversion is described in detail in this post of me: https://github.com/rytilahti/python-miio/issues/929#issuecomment-787442355

However it is not so trivial (as discussed in the linked post):
If we would now split up in seperate entities, that would result in far more update calls to the devices probably causing lots more connection issues with the device due to them beeing overrun with update requests. (Now all parameters are updated in a single call to the device). Therefore an UpdateCoordinator is needed. But for that, the device needs to be set up in __init__.py (to share the UpdateCoordinator between the devices), that means the model needs to be identified already in the __init__.py file and all that logic needs to be shifted from the platform files to the __init__.py. Actually what we want to do is shift that logic to the upstream python-miio library (see linked post).

So basically first the upstream python-miio lib needs a major rework to do the model -> device logic in the python-miio lib.

Reply:
So basically spliiting up in seperate entities is beeing worked on, but is out of scope for this PR (it would get way to huge).

Reply:
Ok, good. I wasn't sure about the roadmap. :+1: